io.github.dida_code.OpstaKultura.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.dida_code.OpstaKultura",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "opstakultura",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "opstakultura",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/dida-code/opstakultura",
+                    "tag" : "v1.1.0"
```

Comment:
This tag doesn't have this commit https://github.com/dida-code/opstakultura/commit/563def9d3e3a839edf211c7352f62db99313c857

Also, you forgot the screenshots, replace them with https://raw.githubusercontent.com/dida-code/opstakultura/v1.1.0/data/assets/Screenshot1.png and https://raw.githubusercontent.com/dida-code/opstakultura/main/data/assets/Screenshot2.png

Reply:
This tag doesn't have this commit https://github.com/dida-code/opstakultura/commit/563def9d3e3a839edf211c7352f62db99313c857
ok I understand you, but can I edit the tag itself?

Reply:
Create a new tag, 1.1.1.

Reply:
I made a tag, I hope it will be ok now, thank you for your help